eth_config.yml

```diff
@@ -0,0 +1,263 @@
+endpoints:
+  - ws://localhost:8545
+
+stale_timeout_seconds: 180 # rotate endpoint if no new blocks for 3 minutes
+
+eth_rpc: true
```

Comment:
we need to avoid chain specific configurations like this
if we really need a global flag, make it generic like

```
flags:
  eth: true
```

Reply:
or we make cli to accept either substrate or eth config?

Reply:
I want this to be a generic JSON RPC gateway so it should work in other context as well